src/routes/event/[username]/[eventId]/+page.svelte

```diff
@@ -63,15 +65,15 @@
 			</div>
 		{:else}
 			<div>
-				<p class="large">{data.overview.dateCreated}</p>
+				<p class="large">{starDate}</p>
 				<p class="small">Start Date</p>
 			</div>
 		{/if}
 		<div>
 			{#if !data.overview.price}
 				<p class="large">Free</p>
 			{:else}
-				<p class="large">USD {data.overview.price}</p>
+				<p class="large">{Number(data.overview.price).toFixed(2)} FLOW</p>
```

Comment:
@IgnacioDebat there is a component called Currency on the component library to render currency amounts